web/src/main/webapp/app/webcert.scss

```diff
@@ -1,9 +1,12 @@
 @CHARSET "UTF-8";
 
 @import "mixins/wc-variables";
-@import "mixins/bootstrap-variables-override"; // Bootstrap override variables must be loaded before bootstrap due to its !default flags
-@import '../bower_components/bootstrap-sass/assets/stylesheets/bootstrap.scss'; // Load bootstrap defaults
-@import "mixins/bootstrap-css-override"; // CSS Override only, no bootstrap variables
+@import "mixins/bootstrap-variables-override";
```

Comment:
Better move the comment so it sits above the row the comment is for (or return it to the same line).

Reply:
Comments now above the lines they comment